src/ch/imedias/rsccfx/view/RsccRequestView.java

```diff
@@ -64,8 +63,8 @@
   private final KeyUtil keyUtil;
 
   Button reloadKeyBtn = new Button();
+  final Button disconnectBtn = new Button();
 
-  private Pane emptyPane = new Pane();
```

Comment:
It looked the same without the empty pane. I had an issue that the empty Pane overlapped my button, so I asked Francois and he said I can delete it